20121228 bw.aljex@gmail.com
Fix dynamic build

--- pianobarfly/Makefile	2012-12-28 05:46:43.954944543 -0500
+++ pianobarfly_bkw/Makefile	2012-12-28 07:14:37.321093753 -0500
@@ -107,7 +107,7 @@
 pianobarfly: ${PIANOBAR_OBJ} ${PIANOBAR_HDR} libpiano.so.0
 	@echo "  LINK  $@"
 	@${CC} -o $@ ${PIANOBAR_OBJ} ${LDFLAGS} -lao -lpthread -lm -L. -lpiano \
-			${LIBFAAD_LDFLAGS} ${LIBMAD_LDFLAGS} ${LIBGNUTLS_LDFLAGS}
+			${LIBFAAD_LDFLAGS} ${LIBMAD_LDFLAGS} ${LIBGNUTLS_LDFLAGS} ${LIBID3TAG_LDFLAGS}
 else
 pianobarfly: ${PIANOBAR_OBJ} ${PIANOBAR_HDR} ${LIBPIANO_OBJ} ${LIBWAITRESS_OBJ} \
 		${LIBWAITRESS_HDR}
@@ -209,7 +209,7 @@
 install: pianobarfly
 endif
 	install -d ${DESTDIR}/${BINDIR}/
-	install -m755 pianobarfly ${DESTDIR}/${BINDIR}/
+	install -s -m755 pianobarfly ${DESTDIR}/${BINDIR}/
 	install -d ${DESTDIR}/${MANDIR}/man1/
 	install -m644 contrib/pianobarfly.1 ${DESTDIR}/${MANDIR}/man1/
 
@@ -221,5 +221,6 @@
 	install -m644 libpiano.a ${DESTDIR}/${LIBDIR}/
 	install -d ${DESTDIR}/${INCDIR}/
 	install -m644 src/libpiano/piano.h ${DESTDIR}/${INCDIR}/
+	ldconfig
 
 .PHONY: install install-libpiano test debug all
